pipeline/Dockerfile

```diff
@@ -23,7 +23,7 @@ COPY --chown=opam pipeline/. .
 RUN sudo chown opam .
 
 # Build the project.
-RUN opam exec -- dune build --profile=release bin/main.exe
+RUN opam exec -- dune build --profile=release ./pipeline/bin/main.exe
```

Comment:
I believe the [CI build is failing](https://deploy.ci3.ocamllabs.io/job/2022-01-17/165555-ocluster-build-402cab) because of this path change?

```

Step 8/20 : RUN opam exec -- dune build --profile=release ./pipeline/bin/main.exe
 ---> Running in f11d5059f3c5
Error: Don't know how to build ./pipeline/bin/main.exe
The command '/bin/sh -c opam exec -- dune build --profile=release ./pipeline/bin/main.exe' returned a non-zero code: 1
```

On line 22 there is a `COPY --chown=opam pipeline/. .` that already takes care of "hiding" the `pipeline/` directory (so either the COPY or the dune build should handle the pipeline dir, but not both?)

Reply:
So the problem here is that the deployer needs absolute path and not relative path. So we have to explicitly say that the path is `pipeline/bin/main.exe`. It's a bit convoluted. Perhaps these changes should live on the live branch? And we can have a github action to sync them? I don't know if this makes sense.

Reply:
Or can we do some environment variable magic to detect that we are in "deployer mode" to use a different path? I don't know enough about deployer, and I'm just thinking out loud here.